sources/packages/forms/src/form-definitions/designationagreementdetails.json

```diff
@@ -3012,6 +3018,32 @@
               "hideLabel": true,
               "tags": []
             },
+            {
+              "key": "officerDuplicateWarningHtml",
+              "label": "HTML",
+              "input": false,
+              "tag": "p",
+              "attrs": [
+                {
+                  "value": "",
+                  "attr": ""
+                }
+              ],
+              "className": "alert alert-warning fa fa-exclamation-triangle w-100",
+              "content": "You have the same official listed as both an Eligibility Officer and an Enrolment Officer. StudentAid BC policy is that these roles must be effectively segregated such that no employee is appointed to perform both duties. An exemption may be granted upon written request and subject to Ministry approval, if a designated institution had fewer than 10 student borrowers who received a combined total of under $50,000 in student financial assistance in the previous program year.\nPlease email DESIGNAT@gov.bc.ca if you want to request an exemption.",
+              "type": "htmlelement",
+              "hideLabel": false,
+              "tags": [],
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+              },
+              "properties": {},
+              "customClass": "banner-warning",
+              "lockKey": true,
+              "customConditional": "let repeat = false;\ndata.dynamicData.enrolmentOfficers.forEach(enrolment => {\n    if (enrolment.name !== \"\") {\n        data.dynamicData.eligibilityOfficers.forEach(eligibility => {\n            if (enrolment.name === eligibility.name) {\n                repeat = true;\n            }\n        });\n    }\n});\nshow = (data.isBCPrivate && repeat);"
```

Comment:
Can the condition be adjusted as below (or something similar) to include the below benefits?
Benefits
1 - Do not execute the `forEachs` if not a BC Private';
2 - If `show = true` stop checking and `return`;
3 - Make the comparison case insensitive.

```js
show = false;
if (!data.isBCPrivate) {
  return;
}
data.dynamicData.enrolmentOfficers.forEach((enrolment) => {
  if (enrolment.name !== "") {
    data.dynamicData.eligibilityOfficers.forEach((eligibility) => {
      if (enrolment.name.toLowerCase() === eligibility.name.toLowerCase()) {
        show = true;
        return;
      }
    });
  }
});
```

Reply:
The check for BC private may be completely eliminated since the entire container is only displayed for BC Privates.

Reply:
updated